packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -67,6 +74,12 @@ const QuickStrategyForm = ({
                                             onChangeDropdownItem('symbol', value, setFieldValue);
                                         }}
                                         onScrollStop={() => onScrollStopDropdownList('symbol')}
+                                        leading_icon={
+                                            is_mobile &&
```

Comment:
Nice! I think we can include this in desktop view as well. References: https://app.zeplin.io/project/5ccc037799118d67c6dc5ba8/screen/5d6e1910b96ae67d82f31f27